test/release/examples/primers/sparse.chpl

```diff
@@ -25,7 +25,7 @@ config var n = 9;
 // see, it will also be useful for operations that want to treat our
 // sparse domain as though it was a dense ``n x n`` set of values.
 //
-const dnsDom = {1..n, 1..n};
+const dnsDom = {0..#n, 0..#n};
```

Comment:
As mentioned on chat, I feel inclined to just leave this primer alone rather than changing it as part of this PR (assuming there isn't a reason that it needs to change that I'm missing).  Specifically, since it doesn't related to the LinearAlgebra module, but sparse domains/arrays in Chapel in general, it doesn't seem linked.  And I think there's benefit in demonstrating that Chapel domains and arrays can use any index set the user wants (where some users are still adamantly 1-based).  Finally, I think that using 0-based makes this code slightly more complex ("what is `#`?"  Lots of `n-1` expressions) in ways that don't relate to the focus of the primer.

If the primer doesn't already, I'd be very open to inserting a sidebar that said something like "Note that Chapel supports arbitrary index sets such that you can create 0-based sparse arrays, -10-based sparse arrays, etc.", potentially with a short example.  But I'd leave the lion's share of the primer untouched relative to what it was before.

Reply:
All good points.  I reverted back to the original 1-based primer.